blocks/variables_dynamic.ts

```diff
@@ -141,46 +141,47 @@ const CUSTOM_CONTEXT_MENU_VARIABLE_GETTER_SETTER_MIXIN = {
   /**
    * Called whenever anything on the workspace changes.
    * Set the connection type for this block.
-   * @param {AbstractEvent} _e Change event.
-   * @this {Block}
    */
-  onchange: function(_e) {
+  onchange: function(this: CustomContextMenuVariableBlock, _e: AbstractEvent) {
     const id = this.getFieldValue('VAR');
     const variableModel = Variables.getVariable(this.workspace, id);
+    if (!variableModel) {
+      return;
+    }
```

Comment:
This is an actual functional change.  Would it not suffice to have written the previous line as
```TS
    const variableModel = Variables.getVariable(this.workspace, id)!;
```
?
Or is there a real bug that you're fixing?